app/src/pages/inside/common/selectedItems/selectedItems.scss

```diff
@@ -3,6 +3,8 @@
   padding-top: 15px;
   padding-left: 15px;
   padding-right: 15px;
+  word-break: break-all;
+  word-break: break-word;
```

Comment:
Let's use `@supports` directive to make sure every browser will use correct style:
```
  word-break: break-all;

  @supports (word-break: break-word) {
    word-break: break-word
  }
```

Reply:
replaced it as suggested